.github/workflows/publish-v2-snapshot.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish v2 snapshots to the Maven Central repository. Runs on merge to v2.
+on:
+  pull_request:
+    branches:
+      - v2
```

Comment:
I don't think we want this, on each PR, we just want the merged PR

```suggestion
```

Reply:
Just there to test for now!

Reply:
so you can remove it now ;)